google/cloud/storage/benchmarks/benchmark_utils.cc

```diff
@@ -177,9 +177,13 @@ void PrintOptions(std::ostream& os, std::string const& prefix,
     os << "\n# " << prefix
        << " Grpc Endpoint: " << options.get<EndpointOption>();
   }
+  if (options.has<gcs::ConnectionPoolSizeOption>()) {
+    os << "\n# " << prefix << " Connection Pool Size: "
```

Comment:
can we say "REST Connection Pool Size: "? so it's obvious the option doesn't apply for the gRPC case.

Reply:
Fixed and added the tests